### ISSUE TITLE
There was an error communicating with your Plex Server.

### ISSUE THREAD

**Issue by OmgImAlexis** [Tautulli/Tautulli#902]
**Version:** 1.4.14

**Branch:** Master

**Commit hash:** 86a9230da8a9ab1af18eacb07a9fd7086437305c

**Operating system:** UNRAID

**Python version:** 2.7.12

**What you did?** Started PlexPy

**What happened?** It showed me an error on the homepage.

**What you expected?** It to connect to Plex.

**How can we reproduce your issue?** Not sure

**What are your (relevant) settings?** All default as far as I can remember

**Link to logs:**
https://gist.github.com/OmgImAlexis/b73860a8bf6e7eaacccd783c4085a8e8

I've had my UNRAID server offline for the last few months, I turned it back on today and updated all of my docker containers and now PlexPy won't show anything from my server on the homepage yet it connects fine in the settings. I'm running Plex Version 2.10.10 on the PlexPass channel.

![screen shot 2016-11-11 at 3 46 52 pm](https://cloud.githubusercontent.com/assets/6525926/20205190/bd5d34ea-a826-11e6-8a16-64b8bf94345a.png)
![screen shot 2016-11-11 at 3 46 42 pm](https://cloud.githubusercontent.com/assets/6525926/20205191/bd5d7df6-a826-11e6-998a-215e3c358026.png)

**Comment by Hellowlol** [comment]
Isn't a lot to go on in the logs. We should log the response code from plex. As there is no reason why it fails

**Comment by JonnyWong16** [comment]
@Hellowlol There is no response code. It's an `IOError` https://github.com/JonnyWong16/plexpy/blob/dev/plexpy/http_handler.py#L89

Is "Use SSL" checked?
https://github.com/JonnyWong16/plexpy/wiki/Frequently-Asked-Questions-(FAQ)#general-q17

**Comment by OmgImAlexis** [comment]
"Use SSL" seems to be the issue, maybe the message could be changed to signify that's the issue.

Any idea why "Use SSL" would work 4 months ago with the same server setup and not now? I literally turned my server off for a few months and now that it's back on it doesn't work.. 😞

**Comment by JonnyWong16** [comment]
Did you read the FAQ answer?

**Comment by OmgImAlexis** [comment]
It's pfsense then, all good.

**Comment by painfulsedative** [comment]
I'm not currently having any issues, but thought I would post this here for any others that may encounter similar circumstances...

i was having this issue with plexpy stating it couldnt communicate with my server as well, and out of the blue... I restarted my server and it didnt help.  I tried changing the IP addresses around to ensure they would verify and after I changed it and clicked save, it would tell me to verify settings in the bottom right corner... and not save the settings.  I decided to do a restart within the PlexPy webUI and after it came back up, it walked me through the initial configuration settings again, as if i was setting it up anew... after this I am able to communicate with the server again.
